include/deal.II/lac/la_parallel_vector.h

```diff
@@ -346,6 +346,20 @@ namespace LinearAlgebra
              const IndexSet &ghost_indices,
              const MPI_Comm  communicator);
 
+      /**
+       * Initialize vector with @p local_size locally-owned and @p ghost_size
+       * ghost degrees of freedoms.
+       *
+       * @note The created underlying partitioner is set up so that the
+       *   number of locally-owned and ghost degrees of freedom match the given
+       *   parameters. However, an interpretation of the ghost indices is not
+       *   possible.
```

Comment:
Update with information on ascending + 1-to-1 distribution of indices similar to the constructor.